terraform/cloud.gov.tf

```diff
@@ -569,14 +569,38 @@ resource "aws_route53_record" "cdn_broker_delegate" {
   ]
 }
 
+# Start SitesUSA subomains
+/*
+    We are trying to keep this file as clean as possible so we are grouping
+    the subdomains here.  We should probably get our own file or something,
+    we have about 46 domains.
+ */
```

Comment:
oh sweet! Not a devops guy here, so I'll have some questions when I actually do this.  But for now I just need to test that this works.